modules/exploits/windows/http/manageengine_adaudit_plus_authenticated_rce.rb

```diff
@@ -438,17 +447,23 @@ def exploit
     # time to trigger the payload
     if @domain_alias.nil?
       # this means check didn't run, so we need to obtain the configured active directory domains
-      target_check_status, target_check_msg, target_check_res = adaudit_plus_target_check
-      unless target_check_status == adaudit_plus_status::SUCCESS
+      target_check_results = adaudit_plus_target_check
+      target_check_msg = target_check_results['message']
+      unless target_check_results['status'] == adaudit_plus_status::SUCCESS
         print_error('Failed to obtain the configured Active Directory domain aliases')
         fail_with(Failure::Unknown, target_check_msg)
       end
+      target_check_res = target_check_results['res']
```

Comment:
Think this is meant to be `server_response`. There is no `res` field in the hash as far as I can tell. This may be a typo from the looks of things.

Reply:
Fixed with 10d0d696f8fd04d0916b375d628993222aa51ec6